Description: Use `patch` instead of git
 Applying patches via won't work with the current setup
 due to submodule paths.
 .
 hyprland (0.40.0-1ppa1) noble; urgency=medium
 .
   * Initial release. Follows release 0.40.0.
Author: Constantin Piber <cp.piber@gmail.com>

---
The information above should follow the Patch Tagging Guidelines, please
checkout https://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: (upstream|backport|vendor|other), (<patch-url>|commit:<commit-id>)
Bug: <upstream-bugtracker-url>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: (no|not-needed|<patch-forwarded-url>)
Applied-Upstream: <version>, (<commit-url>|commit:<commid-id>)
Reviewed-By: <name and email of someone who approved/reviewed the patch>
Last-Update: 2024-05-31

--- hyprland-0.40.0.orig/subprojects/wlroots-hyprland/patches/apply.sh
+++ hyprland-0.40.0/subprojects/wlroots-hyprland/patches/apply.sh
@@ -4,7 +4,7 @@
 PATCHES=$(find patches/ -type f -name '*.patch')
 
 apply () {
-	git apply $PATCH
+	patch -p1 < $PATCH
 }
 
 check_applied () {
